app/prism/Prism.scala

```diff
@@ -31,6 +31,18 @@ object Prism {
 
   def getInstances(stackStageApp: SSA)(implicit config: AMIableConfig, ec: ExecutionContext): Attempt[List[Instance]] = {
     val url = instancesUrl(stackStageApp, config.prismUrl)
+    getInstancesFromUrl(url)
+  }
+
+  def imageUsage(imageOpt: Option[AMI])(implicit config: AMIableConfig, ec: ExecutionContext): Attempt[List[Instance]] = {
+    for {
+      image <- Attempt.fromOption(imageOpt, AMIableErrors(AMIableError("Could not find image", "Could not find image", 404)))
```

Comment:
I am not sure if those messages are the best?...and also since image is an `Option`, would it make sense trying to print its id?